tabular/src/autogluon/tabular/models/tabular_nn/hyperparameters/parameters.py

```diff
@@ -99,9 +100,9 @@ def get_quantile_hyper_params(framework):
     """ Parameters that currently can be searched during HPO """
     hyper_params = get_hyper_params(framework)
     new_hyper_params = {
-        'gamma': 5.0,  # margin loss weight which helps ensure noncrossing quantile estimates
-        # Options: range(0.1, 10.0)
-        'alpha': 0.01,  # used for smoothing huber pinball loss
+        'gamma': 1.0,  # margin loss weight which helps ensure noncrossing quantile estimates
+        'num_layers': 3,
+        'hidden_size': 128,
```

Comment:
From a practical side I haven't seen MLP performing better with 3 layers vs 2 layers. hidden_size can be larger. What's more important for MLP: 1) the order of the components in the layers, 2) training schedule; 3) normalization.

also why not using different sizes of layers? [200, 100]